### ISSUE TITLE
[ATK] Udo Neuroth 68020 Turbo-Karte fails Memory Test

### ISSUE THREAD

**Issue by SukkoPera** [keirf/Amiga-Stuff#40]
I have an A2000 rev 6.2. with an [Udo Neuroth](http://amiga.resource.cx/exp/mtec520) Turbo-Karte, that is an accelerator with a 020 and 4 MB of Fast 32-bit RAM.

Contrary to that the above page says, the RAM does NOT autoconfig, it needs a tool from the install floppy to be run (`Add32Bit`), which will instantly "insert" the RAM into the system. I also have an Oktagon card with 4 MB of Fast RAM, which gets autoconfig'ed at $200000-$5fffff, so the RAM from the Udo Neuroth gets mapped at $600000-$9fffff.

The system appears to be working correctly, though I haven't done extensive testing on it. I have tried to run the ATK Memory Test and:
- If I boot it straight away, it won't see the ram from the Udo Neuroth Card (which is expected AFAIK).
- If I run the "manual" memory scan it will detect 5 MB of Fast RAM, that is the 4 MB from the Oktagon + 1 MB from the Udo Neuroth, and it will all test fine.
- If I run the WB version of ATK once WB has booted and the memory has been configured by the tool, it will see the whole 8 MB (4+4) but the test will fail as soon as it checks the $600000-$67ffff range.

I have tried to edit the start/end addresses to test 1 MB from the Udo Neuroth at a time and this way it will always succeed. So I guess what's happening is that the memory is wrapping at every 1 MB, but I don't understand why this is happening. Other memory tests I have tried either do not see the Udo Neuroth RAM or detect no problems (but they might be crappy and not detect "wrapovers"). Could this be a bug in ATK or more likely a problem with my card?

The card has a "1/4 MB" jumper, but I'm not sure which setting corresponds to the jumper open or closed and the manual does not help. All I can say is that it is currently closed and if I open it the RAM config tool will crash, so it looks like it's in the 4 MB position.

**Comment by SukkoPera** [comment]
After a lot of testing, I can conclude that ATK is right and the board does not have 4 MB. In fact, a look at the board revealed that it has 8 [Siemens HYB514256BJ-70 chips](https://www.alldatasheet.com/datasheet-pdf/pdf/45244/SIEMENS/HYB514256BJ-70.html), which are 256Kx4, thus the total amount is 1 MB. I guess these could be replaced with some other chips (1Mx4?) for the 4 MB configuration.

I can also guess that the 1/4 MB jumper is not supposed to be changed by the user to decide how much RAM they want to see, but it's rather a factory setting (at least if you have the 1 MB version).

Bottom line is there's nothing wrong in ATK, sorry for the waste of time.
